src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/_WelcomeWizard.cshtml

```diff
@@ -112,20 +112,39 @@
                         <legend class="heading-small">Are you going to sign the agreement?</legend>
                         <p>This allows you to spend funds.</p>
                         <label class="block-label selection-button-radio" for="add-agreements-1" data-target="guide-wizard-step-agreements">
-                            <input id="add-agreements-1" type="radio" name="agreements-group" value="1">
+                            @{int value = 1;string check = ""; if (Model.Data.RequiresAgreementSigning > 0) { value = 1; } else { value = 2; check = "checked"; } }
+                            <input id="add-agreements-1" type="radio" name="agreements-group" value="@value" checked=@check>
                             Yes
                         </label>
-                        <label class="block-label selection-button-radio" for="add-agreements-2">
+                        <label class="block-label selection-button-radio" for="add-agreements-2" data-target="value-setup">
                             <input id="add-agreements-2" type="radio" name="agreements-group" value="2">
+
                             No, someone else will sign the agreement
                         </label>
-                        <div class="panel panel-border-narrow js-hidden" id="guide-wizard-step-agreements">
-                          @Model.Data.RequiresAgreementSigning unsigned agreement(s). <a href="@Url.Action("Index", "EmployerAgreement")">View agreements now</a>.
-                        </div>
+                        @if (Model.Data.RequiresAgreementSigning > 0)
```

Comment:
Can we set a "RequiresAgreementSigning" boolean and reuse throughout rather than duplicating count logic